.github/resources/codeql-config.yml

```diff
@@ -0,0 +1,10 @@
+name: CodeQL Config
+
+queries:
+  - uses: security-and-quality
+
+paths:
+  - src
+paths-ignore:
+  - src/node_modules
+  - '**/*.test.js'
```

Comment:
Probably being dumb 😢 but this path also confuses me. `ts-jest` transpiles our tests in memory, but I don't remember ever seeing a "*.test.js" file on disk 🤔

Reply:
Another one that I don't recall, Let's give it a spin and remove it :yum:

Reply:
removed!